nodejs/week1/src/backend/app.js

```diff
@@ -1,3 +1,8 @@
+const app = require("./app");
+
+app.listen(3000, () => {
+    console.log("Example app listening on port 3000!");
+});
```

Comment:
Actually, let this be separated in the server.js. Revert this change and you can merge the PR.